firefox_ui_tests/remote/security/test_ssl_status_after_restart.py

```diff
@@ -0,0 +1,123 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import Wait
+from marionette_driver.errors import NoSuchElementException
+
+from firefox_ui_harness.decorators import skip_if_e10s, skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSSLStatusAfterRestart(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_data = [
+            {
+                'url': 'https://ssl-dv.mozqa.com',
+                'identity': '',
+                'type': 'verifiedDomain'
+            },
+            {
+                'url': 'https://ssl-ev.mozqa.com/',
+                'identity': 'Mozilla Corporation',
+                'type': 'verifiedIdentity'
+            },
+            {
+                'url': 'https://ssl-ov.mozqa.com/',
+                'identity': '',
+                'type': 'verifiedDomain'
+            }
+        ]
+
+        # Set browser to restore previous session
+        self.prefs.set_pref('browser.startup.page', 3)
+
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def tearDown(self):
+        try:
+            self.identity_popup.close(force=True)
+            self.browser.tabbar.close_all_tabs([self.browser.tabbar.tabs[0]])
+        except NoSuchElementException:
+            # TODO: A NoSuchElementException may be thrown here when the test is skipped
+            # as under xvfb.
+            pass
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    @skip_if_e10s
+    @skip_under_xvfb
+    # Bug 995801: Test certificate status after browser restart
+    def test_ssl_status_after_restart(self):
+        for item in self.test_data:
+            with self.marionette.using_context('content'):
+                self.marionette.navigate(item['url'])
+            self.verify_certificate_status(item)
+            self.browser.tabbar.open_tab()
+
+        '''
+        self.restart()
+
+        i = 0
+        for item in self.test_data:
+            self.browser.tabbar.switch_to(i)
```

Comment:
tabbar's `switch_to` claims and looks like it will handle an index. It wraps `self.browser.tabbar.tabs[index].switch_to()` which is easy enough to use here instead.